### PR TITLE
[FLINK-20570][docs] Fix the `NOTE` tip style in process_function page

### PR DESCRIPTION
**Pull request by shizhengchao**
## What is the purpose of the change
Fix the `NOTE` tip style in process_function page:
in `/docs/stream/operators/process_function.md`, line 252.  The `NOTE` css style is different from the others.

## Brief change log
1. Change the `**NOTE:**` to `<span class="label label-info">Note</span>`
2. remove the `{% top %}` 

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: ( no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (no)

### CODE REVIEW

File: docs/dev/stream/operators/process_function.md [hunk 1]
```diff
@@ -246,10 +246,8 @@ class CountWithTimeoutFunction extends KeyedProcessFunction[Tuple, (String, Stri
 </div>
 </div>
 
-{% top %}
-
 
-**NOTE:** Before Flink 1.4.0, when called from a processing-time timer, the `ProcessFunction.onTimer()` method sets
+<span class="label label-info">Note</span> Before Flink 1.4.0, when called from a processing-time timer, the `ProcessFunction.onTimer()` method sets
```
Comment (wuchong) [inline docs/dev/stream/operators/process_function.md:250]: I think `**NOTE:**` is used in more places?

File: docs/dev/stream/operators/process_function.md [hunk 2]
```diff
@@ -246,10 +246,8 @@ class CountWithTimeoutFunction extends KeyedProcessFunction[Tuple, (String, Stri
 </div>
 </div>
 
-{% top %}
```
Comment (wuchong) [inline docs/dev/stream/operators/process_function.md:249]: This is useful to scroll to the page top. You can find the top link is used in many places.

**Review by wuchong — COMMENTED** [review]
I'm not sure about this.

File: docs/dev/stream/operators/process_function.md (hunk 1 above)
Comment (shizhengchao) [reply docs/dev/stream/operators/process_function.md:250]: in process_function.md，`<span class="label label-info">Note</span>`  is used in more places，but `**NOTE:**` is used only place

Comment (shizhengchao) [reply docs/dev/stream/operators/process_function.md:250]: > I think `**NOTE:**` is used in more places?
`**NOTE:**` is used only one place in process_function.md,  
and `<span class="label label-info">Note</span>` is used in more places.

Comment (wuchong) [reply docs/dev/stream/operators/process_function.md:250]: got.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**